src/navigation/config.tsx

```diff
@@ -626,6 +626,20 @@ export const portalSheetConfig: PartialNavigatorConfigOptions = {
   }),
 };
 
+export const activitySheetConfig: PartialNavigatorConfigOptions = {
+  options: ({ route: { params = {} } }) => ({
+    ...buildCoolModalConfig({
+      ...params,
+      backgroundOpacity: 0.8,
+      headerHeight: 0,
+      springDamping: 1,
+      topOffset: safeAreaInsetValues.top + 15,
```

Comment:
i think the activity list looks better not fully touching the top so i've added a bit of space, and also background opacity 0.8 looks better than fully 1 opacity so you can see behind a bit